### ISSUE TITLE
re: Observing or Joining Monday meetings

### ISSUE THREAD

**Issue by SMotaal** [chaoss/wg-diversity-inclusion#235]
Hi folks:

Currently, a few of us have been exploring the space in the Node.js Community Committee in the form of a new initiative [#507](https://github.com/nodejs/community-committee/issues/507) aka Improving Odds (for now) which is meant to tackle equal-access matters that could improve the odds for equal participation and contribution.

The efforts are still relatively young though, and involve the support of many who are backing efforts between @SauloNunes and myself.

We'd like to join in as observers to see how our efforts can be mutually beneficial to the shared values and goals.

I am assuming and hope it is okay to be just dropping in shortly today's meeting and look forward to meeting with the amazing folks behind this very inspiring work.

<h1 align=center>🥇</h1>

> **Note**: There <del>is …</del> <ins>was a small discrepancy in the description of the calendar event now fixed.</ins>

**Comment by klumb** [comment]
Saleh,

You are very welcome and we look forward to seeing you. 

Thank you for catching the discrepancy in the calendar text. I have edited it to match the current meeting time of 11:00 am Central Time. 



Cheers, 


Kevin Lumbard, M.S. PMP (he/him/his)
University of Nebraska at Omaha
College of Information Science and Technology
PKI 367

"We build our computers the way we build our cities -- over time, without a plan, on top of ruins" - Ellen Ullman

**Comment by GeorgLink** [comment]
Yes, you are very welcome to join our call.

Thank you for pointing out the discrepancy with the time. I found another and issued a pull request #236 .

As @klumb said, we meet today in 15 minutes. [Join us](https://unomaha.zoom.us/j/720431288)

The chaoss.community/participate page is correct:
> The D&I working group meets every Monday at 11:00am CT (usually 18:00 CET, [check your local time](https://arewemeetingyet.com/Chicago/2019-11-04/11:00/w/CHAOSS%20D%26I%20WG)) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/1MzDk84BL7FfHDxbFxJz39M72V2Hfc5Y6oCPhOl6woxo/edit)

**Comment by SMotaal** [comment]
Great, I already popped in a bit early and discovered I never left :)

Thanks folks!

**Comment by SMotaal** [comment]
@GeorgLink @klumb and (at)A

Thank you so much for last Monday. 

Truth be told, going into meetings as someone trying to talk disabilities in the open source space, you often need to work hard to rebuild the motivation to keep on going. No question, open source is made up of the most ethical, compassionate, generous and open minded people. But without clarity and resource, most find it hard to reach beyond the curtains of the assumed impracticalities to find their motivated and innovative spirit.

So to find this motivating presence here was invigorating.

<details>
<summary>That's basically what I posted right after.</summary>

> Folks… I’ve just join in on a meeting that was full very awesome open source community members and their amazing efforts that we certainly needed to push our initiative forward — https://github.com/chaoss/wg-diversity-inclusion
> 
> What they can help us devise is a framework for rolling out measurable efforts on many focus areas, including D&I, and so they are very welcoming to our efforts where we can mutually contribute to each of our goals.
> 
> Anyone joining this initiative or simply interested is encouraged to try to join the Monday meetings (details in README) (edited)  — [Saleh `Nov04`](https://node-js.slack.com/archives/CMQM6ME1Z/p1572890809002600)

</details>

There is no doubt, I found a place to belong and contribute — maybe with a less typically learning curve and easing-in cuve — and certainly I would love to see the Node.js initiative in the works to grow into a model of what your framework can achieve, for many to follow.

I would like to invite you guys to join our official unofficial [Node Slackers](http://www.nodeslackers.com/) where we usually connect on Node and other things, and where we have our dedicated `#equal-access-initiative` channel.

Thank you, and look forward to joining tomorrow's meeting.
